simapp/README.md

```diff
@@ -16,7 +16,7 @@ in this testnet.
     `simd` binary inside a new `build` directory. The following instructions are run from inside
     the `build` directory.
 2. If you've run `simd` before, you may need to reset your database before starting a new
-    testnet. You can reset your database with the following command: `$ ./simd unsafe-reset-all`.
+    testnet. You can reset your database with the following command: `$ ./simd tendermint unsafe-reset-all`. * If you are using a version earlier than v0.45.3, it will be `$ ./simd unsafe-reset-all`.
```

Comment:
Thanks, @takapi327!

Given that we are using version control, I don't think we need to explain the old way to run `unsafe-reset-all`.
If you are using an older version, you can always find the older command in the changelog or checkout the version tag.

Reply:
@julienrbrt 
Thank you!
You're absolutely right!
Removed description of the old way to run unsafe-reset-all.